test/test_alias.cpp

```diff
@@ -588,4 +588,55 @@ TEST_F(AliasTest, Set_NoAliasForIncompatibleLayout) {
   EXPECT_FALSE(out_tensor.is_alias_of(in_tensor));
 }
 
+TEST_F(AliasTest, DuplicatedOutputs) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  TensorView* in = makeContigConcreteTensor({2, 3, 5});
+  fusion->addInput(in);
+  TensorView* out = add(in, IrBuilder::create<Val>(5.0));
+  fusion->addOutput(out);
+  // duplicated output
+  fusion->addOutput(out);
+  TensorView* out1 = add(in, IrBuilder::create<Val>(1.0));
+  fusion->addOutput(out1);
+  // duplicated output
+  fusion->addOutput(out);
+
+  FusionExecutorCache fec(std::move(fusion));
+  at::Tensor in_tensor = at::randn({2, 3, 5}).cuda();
+  auto inc_5_tensor = in_tensor.add(5.0);
+  auto inc_1_tensor = in_tensor.add(1.0);
```

Comment:
Thanks to @Priya2698 , we don't have to pass in reference outputs as long as `ExpressionEvaluator` can compute them. 

```suggestion
```

Reply:
Oops, I didn't realize we have an overload there. Thanks.